ts/src/header-validator/validate-json.ts

```diff
@@ -1276,39 +1306,40 @@ function aggregatableSourceRegistrationTime(
   return enumerated(ctx, j, AggregatableSourceRegistrationTime)
 }
 
-function warnInconsistentAggregatableKeys(ctx: Context, t: Trigger): void {
-  const triggerDataKeys = new Set<string>()
-
-  ctx.scope('aggregatable_trigger_data', () => {
-    for (const [index, datum] of t.aggregatableTriggerData.entries()) {
-      ctx.scope(index, () => {
-        for (const key of datum.sourceKeys) {
-          triggerDataKeys.add(key)
-
-          if (!t.aggregatableValues.has(key)) {
-            ctx.scope('source_keys', () =>
-              ctx.warning(
-                `key "${key}" will never result in a contribution due to absence from aggregatable_values`
-              )
-            )
-          }
-        }
-      })
-    }
-  })
-
-  ctx.scope('aggregatable_values', () => {
-    for (const key of t.aggregatableValues.keys()) {
-      if (!triggerDataKeys.has(key)) {
-        ctx.scope(key, () =>
-          ctx.warning(
-            'absence from aggregatable_trigger_data source_keys equivalent to presence with key_piece 0x0'
-          )
-        )
-      }
-    }
-  })
-}
+// TODO(apasel422): Update with new AggregatableValue structure.
+// function warnInconsistentAggregatableKeys(ctx: Context, t: Trigger): void {
```

Comment:
Let's keep the function declaration uncommented but leave its body commented using `/* ... */` and continue calling it on line 1426.